app/livechat/server/livechat.js

```diff
@@ -36,3 +36,55 @@ WebApp.connectHandlers.use('/livechat', Meteor.bindEnvironment((req, res, next)
 	res.write(indexHtmlWithServerURL);
 	res.end();
 }));
+
+settings.get('Livechat_screen_sharing_provider', function(_key, value) {
+	if (!value) {
+		return;
+	}
+
+	value = value.toString();
+	let screenSharingProvider = '';
+
+	if (value === 'Cobrowse.io') {
+		screenSharingProvider = 'cobrowse';
+	}
+
+	if (screenSharingProvider === '') {
+		return;
+	}
+
+	let screenSharingProviderTxt = Assets.getText(`livechat/screen-sharing/${ screenSharingProvider }.txt`);
+
+	if (screenSharingProvider === 'cobrowse') {
+		const license_key = settings.get('Cobrowse.io_License_Key');
+		screenSharingProviderTxt = addCobrowseLicenseKey(screenSharingProviderTxt, license_key);
+	}
+
+	WebApp.connectHandlers.use(`/livechat/screen-sharing/${ screenSharingProvider }.js`, Meteor.bindEnvironment((req, res, next) => {
```

Comment:
Why did you implement this endpoint?
Since you will make the file(script) public, why do you need an endpoint to get this file?
I have no idea why do you need this.